src/core/Table/Table.tsx

```diff
@@ -349,7 +352,10 @@ export const Table = <
                   <div {...columnProps} key={columnProps.key}>
                     {column.render('Header')}
                     {!isLoading && data.length != 0 && (
-                      <FilterToggle column={column} />
+                      <FilterToggle
+                        column={column}
+                        ownerDocument={tableRef.current?.ownerDocument}
```

Comment:
Actually, this won't work because it's passed at the same time as the ref as initialized so it's null

Reply:
Fixed and added test